en/asgardeo/mkdocs.yml

```diff
@@ -313,44 +313,64 @@ nav:
   - APIs:
     - APIs - Overview: apis/index.md
     - Authentication: apis/authentication.md
-    - Application management API: apis/application-management.md
-    - API resource management API: apis/api-resource-management-rest-api.md
-    - Branding Preferences API: apis/branding-preference-rest-api.md
-    - OAuth Dynamic Client Registration API: apis/oauth-dcr.md
-    - Identity provider API: apis/idp.md
-    - User management (SCIM 2.0) API: apis/scim2.md
-    - Role management (SCIM 2.0) API: apis/roles.md
-    - Email templates API: apis/email-template.md
-    - Identity governance API: apis/identity-governance.md
-    - User sessions:
-      - Session management API: apis/session.md
-      - Session extension API: apis/extend-sessions.md
-    - Event configuration API: apis/event-configuration.md
-    - Self-service:
-      - TOTP API: apis/register-mfa/totp.md
-      - Backup codes API: apis/register-mfa/backup-code.md
-      - Password recovery API: apis/register-mfa/password-recovery.md
-      - Export user information API: apis/register-mfa/export-user-info.md
-    - Administrators:
-      - Export admin information API: apis/administrators/export-admin-info.md
-    - Consent management API: apis/consent-management.md
-    - Attribute management API: apis/attribute-management.md
-    - Validation API: apis/validation.md
-    - Idle account identification API: apis/idle-account-identification.md
-    - B2B self service management API: apis/onboard-admins-self-service.md
-    - Organization management API: apis/organization-management.md
-    - Organization discovery configuration management API: apis/organization-discovery-config-mgt-rest-api.md
-    - Organization APIs:
-      - Organization APIs: apis/organization-apis/index.md
+    - System APIs:
+      - Admin advisory management API:  apis/admin-advisory-management-rest-api.md
```

Comment:
not required

Reply:
https://github.com/wso2/docs-is/pull/4563/commits/eb753148eb4ef8762733cd878e6c2f1df7255557